backend/azureblob/azureblob.go

```diff
@@ -88,6 +88,10 @@ Leave blank normally. Needed only if you want to use a service principal instead
 
 See ["Create an Azure service principal"](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) and ["Assign an Azure role for access to blob data"](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad-rbac-cli) pages for more details.
 `,
+		}, {
+			Name: "use_env_creds",
```

Comment:
Assuming this is the same thing, we call this `env_auth` elsewhere so can you rename it for consistency?

Eg https://rclone.org/s3/#s3-env-auth